### DISCUSSION TITLE
Error using Component discovery on markdown files

### DISCUSSION THREAD

**Discussion by acgourley** [WeblateOrg/weblate#12809] [Q&A]
I have a directory like:
```
_i18n/en/component1.md
_i18n/en/component2.md
```
So I figure I can use Component Discovery to load them, right? 

Regex: `_i18n/(?P<language>[^/.]*)/(?P<component>[^/]*)\.md`
File format: `markdown`
Component name: `{{component}}`
Monolingual base filename: `_i18n/en/{{ component }}.md`

However I get an error: The following components could not be created
**{'edit_template': ['Editing template is not supported with this file format.']}**

I've tried tons of different values for the monolingual component too but I'm stumped. Any help appreciated 🙏

(I'm using the hosted weblate so it should be latest version)

**Comment by nijel** [comment]
https://github.com/WeblateOrg/weblate/pull/12855 should fix this.
